templates/CRM/Contact/Form/Edit/Website.tpl

```diff
@@ -41,7 +41,9 @@
 <tr id="Website_Block_{$blockId}">
     <td>{$form.website.$blockId.url.html|crmAddClass:url}&nbsp;</td>
     <td>{$form.website.$blockId.website_type_id.html}</td>
+    {if $blockId gt 1}
     <td colspan="3"><a href="#" title="{ts}Delete Website Block{/ts}" onClick="removeBlock('Website','{$blockId}'); return false;">{ts}delete{/ts}</a></td>
```

Comment:
@elisseck This should probably be indented 2 spaces now its within the if

Reply:
Now I just need to remember how to rebase this properly